libraries/core_libs/network/src/tarcap/shared_states/syncing_state.cpp

```diff
@@ -5,8 +5,7 @@
 
 namespace taraxa::network::tarcap {
 
-SyncingState::SyncingState(uint16_t deep_syncing_threshold)
-    : malicious_peers_(300, 50), kDeepSyncingThreshold(deep_syncing_threshold) {}
```

Comment:
`malicious_peers_` should still have maximum size

Reply:
why? wouldnt that just made us open for easier attacks? Is it really realistic that someone tries millions of connections with different IDs that makes malicious_peers create significant memory leak?

Reply:
if there is an cleanup, the we do need to limit size